### ISSUE TITLE
option to filter RawHttpHandler requests

### ISSUE THREAD

**Issue by tobiaszuercher** [wwwlicious/servicestack-seq-requestlogsfeature#17]
hey,
thank you for your fantastic plugin.

i have a little issue and i'm yet not sure how to solve it.
you provide the possibility to filter out certain requests by type. I'm looking at the ServerSentEventsFeature Heartbeat requests.

Our log is spammed with
```
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
HTTP POST /event-heartbeat responded 200 in 0ms.
```

The heartbeats are handled as a rawhttp handler so the `requestType` is always null.

```csharp
        protected bool ExcludeRequestType(Type requestType)
        {
            return ExcludeRequestDtoTypes != null
                   && requestType != null
                   && ExcludeRequestDtoTypes.Contains(requestType);
        }
```
and it will be logged always. I'm not really interested in all the heartbeats of all clients. What would be an appropriate way to filter it out? Maybe an additional string filter/regex property to filter certain urls out?

**Comment by wwwlicious** [comment]
@tobiaszuercher thanks, the only way at present to filter them out would be to set a [custom logger implementation](https://github.com/wwwlicious/servicestack-seq-requestlogsfeature/blob/master/src/ServiceStack.Seq.RequestLogsFeature/SeqRequestLogsSettings.cs#L94)

This isn't ideal so perhaps it would be better to include a delegate to allow for this kind of filtering.

**Comment by wwwlicious** [comment]
I've added the ability to filter logging per request. To use add the following to the feature init code
```
new SeqRequestLogsFeature(new AppSettings())
{
  SkipLogging = request => request.RawUrl == "/event-heartbeat",
}
```

This is now available from nuget version [4.0.86](https://www.nuget.org/packages/ServiceStack.Seq.RequestLogsFeature/4.0.86) onwards

**Comment by tobiaszuercher** [comment]
thank you so much for this!
